src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.targets

```
@@ -151,14 +151,22 @@
       <_ToolchainPrefix Include="x86_64">
         <ToolPrefix>x86_64-linux-android</ToolPrefix>
       </_ToolchainPrefix>
+      <_ToolName Include="as"/>
+      <_ToolName Include="ld"/>
+      <_ToolName Include="strip"/>
     </ItemGroup>
 
+    <CreateItem
+	Include="@(_ToolchainPrefix)" AdditionalMetadata="Tool=%(_ToolName.Identity)">
```

Does this line need to be indented more?
```suggestion
        Include="@(_ToolchainPrefix)" AdditionalMetadata="Tool=%(_ToolName.Identity)">
```

---

Ah, that's my emacs smuggling TAB inplace of spaces :-)